### PR TITLE
Position slippage

### PR DESCRIPTION
**Pull request by PawelKrawczyk96**

### CODE REVIEW

File: src/containers/NewPositionWrapper/NewPositionWrapper.tsx [hunk 1]
```diff
@@ -262,7 +268,7 @@ export const NewPositionWrapper = () => {
 
         const lowerTick = Math.min(leftTickIndex, rightTickIndex)
         const upperTick = Math.max(leftTickIndex, rightTickIndex)
-
+        console.log('INIT POSITION')
```
Comment (awojciak) [inline src/containers/NewPositionWrapper/NewPositionWrapper.tsx:271]: remove this

File: src/components/NewPosition/NewPosition.tsx [hunk 2]
```diff
@@ -242,7 +280,7 @@ export const NewPosition: React.FC<INewPosition> = ({
             onChangePositionTokens(index1, index2, fee)
           }}
           onAddLiquidity={() => {
-            if (tokenAIndex !== null && tokenBIndex !== null) {
+            if (tokenAIndex !== null && tokenBIndex !== null && poolIndex !== null) {
```
Comment (awojciak) [inline src/components/NewPosition/NewPosition.tsx:283]: what about case when pool index is null, because we're initializing new pool?

**Review by awojciak — CHANGES_REQUESTED** [review]
initializing new pools is not working

**Comment by wojciech-cichocki** [comment]
`Position Transaction Settings` instead of `Swap Transaction Settings`
Screenshot from 2022-03-11 16-45-29

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
I get this error when I try to create pool
Screenshot from 2022-03-11 16-48-20

**Comment by wojciech-cichocki** [comment]
> I get this error when I try to create pool Screenshot from 2022-03-11 16-48-20

now works, I have broken account

**Pull request merged**